app/src/debug/res/xml/authenticator.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<account-authenticator
+    xmlns:android="http://schemas.android.com/apk/res/android"
+
+    android:accountType="org.warmshowers.debug"
+    android:icon="@mipmap/ic_launcher"
+    android:label="@string/label_debug"
```

Comment:
Instead of having two `authenticator.xml` files, I guess we could have `@string/label` defined differently in `main/res/values/strings.xml` and `debug/res/values/strings.xml`. This also requires to change the `ACCOUNT_TYPE` in `app/build.gradle` to
`resValue "string", 'ACCOUNT_TYPE', '"org.warmshowers.debug"'` (see https://stackoverflow.com/a/28954193).
Thoughts?

Reply:
Oh right, different `strings.xml` to get rid of `@string/label_debug` is a good idea.

Changing `ACCOUNT_TYPE` from `buildConfigString` to `resValue` complicates its use in `AccountManager`, as I can't just `getString()` there. But  adding an additional `resValue` would be fine.